desktop/src/main/java/bisq/desktop/main/overlays/windows/NewTradeProtocolLaunchWindow.java

```diff
@@ -0,0 +1,176 @@
+
+import javafx.geometry.HPos;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.jetbrains.annotations.NotNull;
+
+import static bisq.desktop.util.FormBuilder.addLabel;
+
+@Slf4j
+public class NewTradeProtocolLaunchWindow extends Overlay<NewTradeProtocolLaunchWindow> {
+
+
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Public API
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    @Override
+    public void show() {
+        width = 700;
+        hideCloseButton();
+        super.show();
+    }
+
+
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Protected
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    @Override
+    protected void createGridPane() {
+        super.createGridPane();
+        gridPane.setVgap(20);
+        gridPane.getColumnConstraints().get(0).setHalignment(HPos.CENTER);
+        gridPane.setPadding(new Insets(74, 64, 74, 64));
+    }
+
+    @Override
+    protected void addHeadLine() {
+
+        Label versionNumber = new AutoTooltipLabel(BisqEnvironment.DEFAULT_APP_NAME + " v1.2.0");
+        versionNumber.getStyleClass().add("news-version");
+        HBox.setHgrow(versionNumber, Priority.ALWAYS);
+        versionNumber.setMaxWidth(Double.MAX_VALUE);
+
+        Button closeButton = FormBuilder.getIconButton(MaterialDesignIcon.CLOSE);
+        closeButton.setOnAction(event -> hide());
+        HBox.setHgrow(closeButton, Priority.NEVER);
+
+        HBox header = new HBox(versionNumber, closeButton);
+
+        GridPane.setRowIndex(header, ++rowIndex);
+        GridPane.setColumnSpan(header, 2);
+        gridPane.getChildren().add(header);
+
+        Label headlineLabel = addLabel(gridPane, ++rowIndex, headLine);
+        headlineLabel.getStyleClass().add("news-headline");
+        GridPane.setMargin(headlineLabel, new Insets(10, 0, 0, 0));
+        GridPane.setColumnSpan(headlineLabel, 2);
+    }
+
+    @Override
+    protected void addMessage() {
+        createContent();
+    }
+
+    @Override
+    protected void onShow() {
+        display();
+    }
+
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Private
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    private void createContent() {
+        HBox content = new HBox();
+        content.setMinWidth(700);
+        content.setAlignment(Pos.CENTER);
+        content.setSpacing(20);
+
+        VBox accountSigning = getFeatureBox(Res.get("popup.news.launch.accountSigning.headline"),
+                Res.get("popup.news.launch.accountSigning.description"),
+                "image-account-signing-screenshot",
+                "https://docs.bisq.network/payment-methods#account-signing");
```

Comment:
This link is broken

Reply:
@m52go This is a page that will be added, right?